tools/sidekick/app.js

```diff
@@ -361,7 +361,7 @@
   function addEditPlugin(sk) {
     sk.add({
       id: 'edit',
-      condition: (sidekick) => sidekick.isHelix(),
+      condition: (s) => s.config.owner && s.config.repo && s.isHelix(),
```

Comment:
Why not adding the `config.owner` and `config.repo` to `isHelix` ? We give a helper method but still need to know some of the config / internal logic to be able to use it...